can/interfaces/udp_multicast/bus.py

```diff
@@ -320,21 +340,37 @@ def recv(
                 self.max_buffer, self.received_ancillary_buffer_size
```

Comment:
The attribute `self.received_ancillary_buffer_size` does not exist if `self.timestamp_nanosecond == False`

Reply:
Good catch